### PR TITLE
Corrected minor typos in README.md for the display libs

### PR DESCRIPTION
**Pull request by Pinkyatacme**
Typos prevented compilation if used as a base!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,3 +27,5 @@ The table below shows the current state of compatibly and some notes to set expe
 We will keep this information updated as things develop.
 
 **Note:** It's very early days for Raspberry Pi Pico and it's likely that our libraries will undergo quite a lot of changes over the next couple of weeks as we take in feedback and expand the functionality.
+
+**Personal clone**
```
Comment (Gadgetoid) [inline README.md:31]: !?

**Comment by Gadgetoid** [comment]
Thank you... I think an unwanted line has crept into the bottom of the README. You might want to rebase out that commit.

File: README.md (hunk 1 above)
Comment (Pinkyatacme) [reply README.md:31]: Sorry - incorrect commit sent! Picked up the wrong README.md file.
NOTE TO SELF: Read your commits! ;)

**Comment by Gadgetoid** [comment]
Thank you!

**Pull request merged**